filebeat/module/haproxy/_meta/fields.yml

```diff
@@ -30,24 +38,24 @@
         - name: server_name
           description: server_name is the name of the last server to which the connection was sent.
 
-        - name: time_client_req
-          description: time_client_req is the total time in milliseconds spent waiting for a full HTTP request from the client (not counting body) after the first byte was received.
+        - name: total_waiting_time_ms
+          description: Total time in milliseconds spent waiting in the various queues
           type: long
 
-        - name: time_queue
-          description: time_queue is the total time in milliseconds spent waiting in the various queues.
+        - name: connection_wait_time_ms
+          description: Total time in milliseconds spent waiting for the connection to establish to the final server
           type: long
 
-        - name: time_backend_connect
-          description: time_backend_connect is the total time in milliseconds spent waiting for the connection to establish to the final server, including retries.
+        - name: bytes_read
+          description: bytes_read is the total number of bytes transmitted to the client when the log is emitted.
           type: long
 
-        - name: time_server_response
-          description: time_server_response is the total time in milliseconds spent waiting for the server to send a full HTTP response, not counting data.
+        - name: time_queue
+          description: time_queue is the total time in milliseconds spent waiting in the various queues.
           type: long
 
-        - name: time_duration
-          description: time_duration is the time the request remained active in haproxy, which is the total time in milliseconds elapsed between the first byte of the request was received and the last byte of response was sent.
+        - name: time_backend_connect
+          description: time_backend_connect is the total time in milliseconds spent waiting for the connection to establish to the final server, including retries.
```

Comment:
If you want to rewrite these descriptions on these PRs do it, if not we can do it in another PR, but we should do it for consistency at some moment.

Reply:
Done in 884ef4f772540389e5c3e96ee8a1a91ef9db36c1

Reply:
Oh sorry, I think I didn't explain well, I meant the other way around :slightly_frowning_face: If you look at other modules they don't start with the name of the field.

Reply:
I don't know, I have spent some time digging in the doc without really understanding what it is. I was supported on the idea that if it's correct in the Grok patterns in that way, it is for us. I really don't care about placing it outside on inside of the default format.

So if you can clarify this, just tell me where you'd like that I place it and I'll do it :)